packages/ledger-icp/candid/index.d.ts

```diff
@@ -74,6 +74,17 @@ export type Operation =
         spender: [] | [string];
       };
     };
+export interface SettledTransaction {
+  memo: bigint;
+  icrc1_memo: [] | [Uint8Array | number[]];
+  operation: Operation;
+  timestamp: TimeStamp;
+  created_at_time: [] | [TimeStamp];
+}
+export interface SettledTransactionWithId {
```

Comment:
How about adding extra imports in `index.ts` to also export these with the original names `Transaction` and `TransactionWithId` to making updating less painful?

Reply:
It's a draft PR. Good idea.

Reply:
Indeed 👍

Reply:
I'm discussing with Mathias. There has not yet been a release with this candid and we might want to move the field to a different type before there is a release. So things might still be in flux.